front-end/src/components/Table.jsx

```diff
@@ -1,22 +1,22 @@
 import React from 'react'
 import styled from 'styled-components'
-import { latestData, orderedHeaders } from './DataManagement'
+import { latestData, orderedHeaders } from '../data/DataManagement'
 
 const PrettyTable = styled.table`
-`
+`;
```

Comment:
tog vi inte bort dessa ? :)

Reply:
Har missat denna, la bara in att den inte kräver semikolon, men man får fortfarande ha det